manifests/e/ETHZurich/SafeExamBrowser/3.7.0/ETHZurich.SafeExamBrowser.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: ETHZurich.SafeExamBrowser
+PackageVersion: 3.7.0
+InstallerType: burn
+Installers:
+- Architecture: neutral
+  InstallerUrl: https://github.com/SafeExamBrowser/seb-win-refactoring/releases/download/v3.6.0/SEB_3.6.0.633_SetupBundle.exe
+  InstallerSha256: B65740B154016E5875F280BDA7D59DE0FA50EE4B29CD719D23925C7C24999F24

```

Hi @PckgrBot,

The `InstallerUrl` has the previous version. 

```suggestion
  InstallerUrl: https://github.com/SafeExamBrowser/seb-win-refactoring/releases/download/v3.7.0/SEB_3.7.0.682_SetupBundle.exe
  InstallerSha256: B4B4F2BFF48F5CDB9C33B4B81D2607B7A0F0355726E9F9BD8070653C1F178B52
```